tools/terraform/ecs.tf

```diff
@@ -230,20 +265,16 @@ resource "aws_ecs_task_definition" "backend" {
             value = var.logging_debug
           },
           {
-            name  = "FLEET_LOGGING_JSON"
-            value = var.logging_json
-          },
-          {
-            name  = "FLEET_S3_BUCKET"
-            value = aws_s3_bucket.osquery-carve.bucket
+            name  = "FLEET_REDIS_MAX_IDLE_CONNS"
+            value = "100"
           },
           {
-            name  = "FLEET_S3_PREFIX"
-            value = "carve_results/"
+            name  = "FLEET_REDIS_MAX_OPEN_CONNS"
+            value = "100"
           },
           {
-            name  = "FLEET_LICENSE_KEY"
-            value = var.fleet_license
+            name  = "FLEET_OSQUERY_ASYNC_HOST_REDIS_SCAN_KEYS_COUNT"
+            value = "10000"
```

Comment:
we're removing file carving and json logging? I agree with adding the new redis configs though.